lib/server.js

```diff
@@ -250,7 +250,7 @@ export default class Server {
       @return {Boolean}
       @public
     */
-    this.logging = config.logging !== undefined ? this.logging : undefined;
+    this.logging = config.logging || undefined;
 
     this.testConfig = this.testConfig || undefined;
```

Comment:
I haven't looked at this code closely enough.  Why does this kind of line exist?  The line being changed above is different, it's checking for `config`, whereas the line here is checking `this`.